src/pybind/mgr/dashboard/frontend/e2e/dashboard.e2e-spec.ts

```diff
@@ -0,0 +1,223 @@
+import { $$, browser, by, element } from 'protractor';
+import { Helper } from './helper.po';
+
+describe('Dashboard Main Page', () => {
+  let dashboard: Helper['dashboard'];
+  let daemons: Helper['daemons'];
+  let hosts: Helper['hosts'];
+  let osds: Helper['osds'];
+  let pools: Helper['pools'];
+  let monitors: Helper['monitors'];
+  let iscsi: Helper['iscsi'];
+
+  beforeAll(() => {
+    dashboard = new Helper().dashboard;
+    daemons = new Helper().daemons;
+    hosts = new Helper().hosts;
+    osds = new Helper().osds;
+    pools = new Helper().pools;
+    monitors = new Helper().monitors;
+    iscsi = new Helper().iscsi;
+  });
+
+  afterEach(() => {
+    Helper.checkConsole();
+  });
+
+  describe('Check that all hyperlinks on cells lead to the correct page and fields exist', () => {
+    beforeAll(() => {
+      dashboard.navigateTo();
+    });
+
+    it('should check all linked cells lead to correct page', () => {
+      // Grabs cell and then clicks the hyperlink, some cells require different
+      // methods as stated below
+      const url = 'http://localhost:4200/#/dashboard';
```

Comment:
jenkins uses a different url for the dashboard.
`Expected 'https://ceph-builders:41040/#/dashboard' to equal 'http://localhost:4200/#/dashboard'.`
Maybe you can use `#/dashboard` instead of full url and change the expects to check if the url contains the string.

Reply:
Should be resolved now